documentation/modules/cruise-control/con-cruise-control-overview.adoc

```diff
@@ -11,12 +11,19 @@
 
 Brokers in a Kafka cluster can become unevenly loaded for many reasons.
 For example, partitions that handle large amounts of message traffic can be unevenly distributed across the available brokers.
-Monitoring broker load and reassigning busy partitions to brokers with spare capacity is time consuming and adds to the overhead of managing a Kafka cluster.
-Cruise Control is a tool for automating the monitoring and rebalancing of partitions across a Kafka cluster.
+Monitoring this broker load and reassigning busy partitions to brokers with spare capacity is time consuming and adds to the overhead of managing a Kafka cluster.
+
+Cruise Control helps to automate the monitoring and rebalancing of partitions across a Kafka cluster.
 When deployed alongside a Kafka cluster, Cruise Control can perform the following cluster balancing tasks:
```

Comment:
_can perform_? Or _performs_? Is this by default, automatic or depending on how its configured?

Reply:
_Performs_ works better here. Thanks.

The monitoring part is automatic; the rest relies on user input.